src/styles/blocks/nav.scss

```
@@ -0,0 +1,51 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: $normal-font;
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    font-weight: 500;
+  }
+
+  &__item {
+    margin-right: 20px;
+    position: relative;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    height: 60px;
+    line-height: 60px;
+    color: #000;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-style: normal;
```

not sure why you need `font-style: normal;`?
and you've forgot few important properties.
```suggestion
    font-weight: ?;
    font-size: ?;
```